cmd/embedded-cluster/join.go

```diff
@@ -42,6 +48,7 @@ type JoinCommandResponse struct {
 	EndUserK0sConfigOverrides string    `json:"endUserK0sConfigOverrides"`
 	MetricsBaseURL            string    `json:"metricsBaseURL"`
 	AirgapRegistryAddress     string    `json:"airgapRegistryAddress"`
+	Proxy                     *Proxy    `json:"proxy"`
```

Comment:
omitempty?
```suggestion
	Proxy                     *Proxy    `json:"proxy,omitempty"`
```

Reply:
Isn't this only needed in case we would marshal this also in embedded cluster? We do this within [kots](https://github.com/replicatedhq/kots/pull/4664/files#diff-416168ed19d2f0fac6d14240569d6e03f063d5ee3ffd5ee0dc6237d325d0040dR36).

Reply:
fair enough